AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -536,30 +536,23 @@ - (BOOL)openUrlUsingSharedApp:(NSURL *)url {
 
 - (void)openUrlInAuthenticationSessionOrSafari:(NSURL *)url {
 
-/*
- * Only iOS 9.x and 10.x will download the update after users click the "Install" button. We need to force-exit the application for other
- * versions or for any versions when the update is mandatory.
- */
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wpartial-availability"
-  Class clazz = [SFSafariViewController class];
-  if (clazz) {
-
-    // iOS 11
-    Class authClazz = NSClassFromString(@"SFAuthenticationSession");
+  /*
+   * Only iOS 9.x and 10.x will download the update after users click the "Install" button. We need to force-exit the application for other
+   * versions or for any versions when the update is mandatory.
+   */
+  if (@available(iOS 11.0, *)) {
```

Comment:
Not part of your PR but `SFAuthenticationSession` is deprecated so we have to handle iOS 12+ separately.

Reply:
I think it should be in a separate PR, so just added TODO, is that ok?

Reply:
That works.